sdk/core/azure-core/inc/azure/core/internal/cryptography/sha_hash.hpp

```diff
@@ -33,7 +33,7 @@ namespace Azure { namespace Core { namespace Cryptography { namespace _internal
      * @brief Cleanup any state when destroying the instance of #Sha256Hash.
      *
      */
-    ~Sha256Hash() {}
+    ~Sha256Hash() override = default;
```

Comment:
nit: seems like an unrelated change

Semantically, is this identical to just using `{}`?

Reply:
Yes, it is identical, maybe the default didn't make the gcovr think that it was not covered. I'll have to check.